Dockerfile

```diff
@@ -1,48 +1,13 @@
-# [Choice] Debian version: buster, stretch
-#ARG VARIANT="buster"
-#FROM buildpack-deps:${VARIANT}-curl
+FROM ubuntu:20.04
```

Comment:
you might consider keeping the alpine image due to image size, it's essentially debian linux and it's pretty well supported. Is there an issue with OpENer on alpine that made you switch to ubuntu?